File: core/src/main/java/com/mcreater/amclcore/api/mihoyo/genshin/model/GenshinGachaDataExtraConfigModel.java
package com.mcreater.amclcore.api.mihoyo.genshin.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class GenshinGachaDataExtraConfigModel {
    @SerializedName("reliquary_fst_attr")
    private GenshinGachaDataExtraConfigItemModel reliquaryFstAttr;
    @SerializedName("reliquary_sec_attr")
    private List<GenshinGachaDataLevelModel> reliquarySecAttr;
    @SerializedName("reliquary_type_define")
    private List<GenshinGachaDataLevelModel> reliquaryTypeDefine;
}

File: core/src/main/java/com/mcreater/amclcore/api/mihoyo/genshin/model/GenshinGachaDataExtraConfigModel.kt
package com.mcreater.amclcore.api.mihoyo.genshin.model

import com.google.gson.annotations.SerializedName

data class GenshinGachaDataExtraConfigModel(
    @SerializedName("reliquary_fst_attr") var reliquaryFstAttr: GenshinGachaDataExtraConfigItemModel? = null,
    @SerializedName("reliquary_sec_attr") var reliquarySecAttr: List<GenshinGachaDataLevelModel>? = null,
    @SerializedName("reliquary_type_define") var reliquaryTypeDefine: List<GenshinGachaDataLevelModel>? = null
)